File: src/main/java/com/yash/wall/service/CommentService.java
package com.yash.wall.service;

import java.util.List;

import com.yash.wall.entity.Comment;
import com.yash.wall.vo.CommentDetailsVO;

public interface CommentService {
    public List<CommentDetailsVO> getAllcommentsByBlogId(int blogid);

    public void addCommentToBlog(Comment comment);
}

File: wall-client/src/service/CommentService.ts
import { requestHelper } from './http';

class CommentService {
    async addCommentToBlog(blogid: string, content: string) {
        const path = `comment`
        const body = {
            blogid,
            content
        }
        return await requestHelper('post', path, body)
    }
}


export default CommentService;